ArduCopter/takeoff.cpp

```diff
@@ -54,6 +54,11 @@ void Mode::_TakeOff::start(float alt_cm)
     _running = true;
     take_off_start_alt = copter.pos_control->get_pos_target_z_cm();
     take_off_complete_alt  = take_off_start_alt + alt_cm;
+#if AP_TIE_DOWN_CLAMPS_ENABLED
+    if (copter.option_is_enabled(Copter::FlightOption::TAKEOFF_TIEDOWN_RELEASE)) {
+        copter.landinggear.tie_down_release();
```

Comment:
How about moving this outside of landing gear.  While the tie-down is perhaps physically on the landing gear I'm not sure it's too related is it?